src/Pos/Context/Functions.hs

```diff
@@ -4,6 +4,7 @@ module Pos.Context.Functions
        (
          -- * Genesis
          GenesisUtxo(..)
+       , mkGenesisTxpContext
```

Comment:
@gromakovsky can you explain, why `Context` can re-export `GenesisUtxo`, but `Functions` can't re-export `mkGenesisTxpContext`?